integrations/jenkins/Jenkinsfile

```diff
@@ -97,6 +97,12 @@ pipeline {
             defaultValue: ''
         )
 
+        booleanParam(
+            name: 'SKIP_EXCLUDED',
+            description: 'Enable to skip scopes and paths that have been excluded via configuration.',
```

Comment:
Question: One what granularity does the "excluding paths" work? (file vs. project)

If the latter, would the comment be clearer if it said: "Enable to skip excluded scopes and projects" ?

Reply:
> One what granularity does the "excluding paths" work? (file vs. project)

While of the tools are you referring to specifically?

Reply:
The scanner.

Reply:
> The scanner won't skip excluded paths

For reference, that's https://github.com/oss-review-toolkit/ort/issues/5018 BTW.

> which is why I proposed the alternative comment.

> "Enable to skip excluded scopes, projects and packages." 

As [packages cannot be excluded on their own](https://github.com/oss-review-toolkit/ort/issues/3537) either, this wording has problems as well. I'll think of something else and update the PR.

Reply:
Packages can be excluded by either a scope exclude or by a path exclude matching the project's definition file.

Reply:
> Packages can be excluded by either a scope exclude or by a path exclude

Exactly. So packages are never excluded explicitly, but only implicitly via scopes or via projects. My goal was to not document this "implictness", but focus on the first-party entities that we can explicitly exclude. I'm open for other wordings that consider this.

Reply:
I believe it's clearer to speak about "what" is excluded (package, project) rather than about "how" its being excluded (scope exclude, path exclude). But I'll leave it up to you.

Reply:
> I believe it's clearer to speak about "what" is excluded (package, project)

In terms of ORT configuration, we have "*scope* excludes" and "*path* excludes", so to me "scope" and "path" are the "what" here, as those are the entities the user has to deal with.

While you were approving, I came up with

> Enable to skip scopes and paths (and affected packages / projects) that have been configured to be excluded.

Would that be better than the current wording in your eyes?

Reply:
> Would that be better than the current wording in your eyes?

I'd still have similar comments on the new version, but this is really opinionated. I'm fine with your version as well.

Reply:
I don't have an opinion on that one. A bit similar. But let's just do as you prefer, it's too opinionated I guess

Reply:
I had pushed this [here](https://github.com/oss-review-toolkit/ort/pull/8223) for reference, but decided to close it.